frontend/packages/console-shared/src/components/dropdown/ResourceDropdown.tsx

```diff
@@ -66,6 +66,7 @@ interface ResourceDropdownProps {
   onLoad?: (items: { [key: string]: string }) => void;
   showBadge?: boolean;
   autocompleteFilter?: (strText: string, item: object) => boolean;
+  appendItems?: { [key: string]: string | React.ReactElement };
```

Comment:
can we could extract this as a `ResourceDropdownItems`Type and use it in other places like onLoad for consistency

Reply:
Updated